xtable-core/src/main/java/org/apache/xtable/delta/DeltaSchemaExtractor.java

```diff
@@ -90,6 +90,7 @@ private DataType convertFieldType(InternalField field) {
         return DataTypes.LongType;
       case BYTES:
       case FIXED:
+      case UUID:
```

Comment:
There is some metadata you an add to a field if I am remember correctly, we will want to pack some context about the fact that this is a UUID if possible

Reply:
> There is some metadata you an add to a field if I am remember correctly, we will want to pack some context about the fact that this is a UUID if possible

That's a good catch, I added "xtableLogicalType" in metadata in latest commit